cloud/packet/public/resources/device.go

```diff
@@ -46,6 +46,14 @@ type Device struct {
 	ProjectID        string
 }
 
+func (r *Device) String() string {
```

Comment:
Where's this function being used and what exactly it does?

The PR looks good, thanks for the contribution! I would just like to clarify this before merging.

Reply:
Without the Stringer debug log does not clearly show what value each property of Device has, i can remove if you want, but it makes logs a bit more readable here.

Reply:
I think it's okay to stay for now.